libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -506,7 +506,7 @@ void AP_ADSB::send_dynamic_out(const mavlink_channel_t chan)
     const uint64_t gps_time = gps.time_epoch_usec();
     const uint32_t utcTime = gps_time / 1000000ULL;
 
-    const AP_Baro &baro = _ahrs.get_baro();
+    const AP_Baro &baro = AP::ahrs().get_baro();
```

Comment:
While valid for rote replacement, we have a baro singleton, so why not use that?

Reply:
This is now a subject of a different PR, which uses the Baro singleton everywhere.